libs/src/oracle2/services/gql/utils.ts

```diff
@@ -53,11 +56,13 @@ export function convertV1(
 
 export function convertAssertion(
   assertion: GqlAssertion,
-  chainId: number
+  chainId: number,
+  oracleAddress: string
 ): Assertion {
   return {
     chainId,
-    oracleType: OracleType.Asserter,
+    oracleType: "Optimistic Asserter",
```

Comment:
we should probably change these to OOv3 before we get any more tangled up in this. I need to do the same on the UI type. spoke to chris and we're still going to call the OOv3 output "assertions"

Reply:
yes we do need to do this eventually, and that is confusing